pythia

```diff
@@ -106,6 +106,8 @@ command_build() {
     cleanup_install_dir
 
     make_dist 'python' ${PYTHON_BUILD_DIR}
+
+    build_sftp_batch
```

Comment:
```suggestion
    create_publish_current_build_sftp_batch
```

In this way the name of the function should be self documented.

Reply:
In the end, I felt like adding a comment not only for this, but also for the `make_dist` line above… :-]